go/api/response_handlers.go

```diff
@@ -31,6 +31,15 @@ func handleStringOrNullResponse(response *C.struct_CommandResponse) string {
 	return handleStringResponse(response)
 }
 
+func handleStringOrNullResponseWithFree(response *C.struct_CommandResponse) string {
+	if response == nil {
+		return ""
+	}
+	defer C.free_command_response(response)
```

Comment:
Shouldn't we free memory on handling all responses?

Reply:
I am using the defer to ensure the memory is freed , even if the function returns early .

Reply:
Right, I missed that other handlers already do the same. My mistake!